vcpkg/consume/caching-workload-identity-federation.md

```diff
@@ -0,0 +1,188 @@
+
+## 2 - Set Up Workload Identity Federation with Azure DevOps
+
+>[!NOTE]
+> This step is intended to be identical to the instructions in [Azure DevOps' documentation](/azure/devops/pipelines/release/configure-workload-identity?view=azure-devopstabs=managed-identity&preserve-view=true#create-a-service-connection-for-managed-identity-authentication-in-azure-devops)
+
+In the Azure DevOps portal for the project in which you wish to run Pipelines, select
+"Project Settings" in the lower left corner. Then select "Service Connections" on the left.
+Then choose "New Service Connection" on the right. Choose the "Azure Resource Manager" radio button,
+and press Next. Select "Workload identity federation (manual)" and press next. Name the
+service connection and press next. For purposes of this documentation, the service connection was
+named `vcpkg-docs-identity-connection`. At this point, Azure DevOps should be showing an issuer
+and subject identifier.
+
+In another tab, go to the Azure Portal navigate to the managed identity created in step 1. On the
+left select Settings/Federated Credentials, and select 'Add Credential'. In the drop down, select
+'Other'. Copy the 'Issuer URL' and 'Subject Identifier' from Azure DevOps into the form. Give the
+federated credential a name, and press 'Add'. For purposes of this documentation, the name used was
+`azure-devops-credential`.
+
+## 3 - Assign Managed Identity Permissions to the Storage Account Container
+
+Skip this step if you have already granted your managed identity permissions to your Storage Account
+Container.
+
+In the Azure Portal, go to the page for the Storage Account to use. On the left, select
+Data Storage/Containers, and choose the container you wish to use. This should open the Azure Portal
+to the properties for that specific container. On the left select "Access Control (IAM)", and choose
+"Add -> Add Role Assignment". Select "Azure Blob Data Reader" for read only access, or
+"Azure Blob Data Contributor" for read/write access, and press Next. Then, select the
+"Managed Identity" radio button, and click + Select Members. Select the managed identity you created
+in step 1. Then select 'Review and Assign' twice.
+
+Then, return to the page for the storage account container, and again on the left select
+"Access Control (IAM)", and choose "Add -> Add Role Assignment". Select "Storage Blob Delegator" and
+press Next. Then, select the "Managed Identity" radio button, and click + Select Members. Select the
+managed identity you create in step 1. Then select 'Review and Assign' twice.
+
+## 4 - Generate a SAS token, and use it with Caching
+
+::: zone pivot="shell-cmd"
+>[!NOTE]
+> These instructions are impractical with 'CMD', please choose 'PowerShell' or 'Bash' at the top
+> of the page.
+::: zone-end
+::: zone pivot="shell-powershell, shell-bash"
+In the yml where you wish to invoke vcpkg in Azure DevOps, use the AzureCLI@2 task to mint a SAS

```

Comment:
```suggestion
In the YAML file that defines your ADO pipeline invoking vcpkg, use the AzureCLI@2 task to mint a SAS
```

Reply:
I'm happy with `yml` -> `YAML file`, but I have 2 problems with the rest of this change.

1. I don't want to say 'ADO'; I don't think it's good to abbreviate our own trademarks in documentation, and to me this still causes confusion because there are many things named 'ADO', such as the database access layer ADO "ActiveX Data Objects".
2. This new phrasing saying the pipeline invoking vcpkg implies that there is already a pipeline doing that, but I'm assuming they don't already have a pipeline doing that.

Would you be happy if I only applied the YAML File part?